promises-async-await.adoc

```diff
@@ -1,4 +1,4 @@
-= Node.js - Promises and async/await
+= JavaScript Promises Cheat Sheet
```

Comment:
Changing Node.js to Javascript does make sense here,  but i think we should still mention the async/await part